pkg/chunked/storage_linux.go

```diff
@@ -640,6 +694,13 @@ type hardLinkToCreate struct {
 	metadata *internal.ZstdFileMetadata
 }
 
+func parseBooleanPullOption(storeOpts *storage.StoreOptions, name string, def bool) bool {
+	if value := storeOpts.PullOptions[name]; strings.ToLower(value) == "true" {
+		return true
```

Comment:
Shouldn’t this be `if value, ok := … { return strings.ToLower(value) == "true" }` so that users can explicitly set options to `false`?

(Ultimately, ideally this should probably fail on typos. Or parse the options at store creation time, and fail then? But that’s expanding the scope quite a bit.)